.github/workflows/benchmarks.yml

```diff
@@ -22,6 +22,10 @@ jobs:
       - uses: google/wireit@setup-github-actions-caching/v1
 
       - name: NPM install
+        env:
+          # Force installed chromedriver version to match the available chrome
+          # version regardless of package-lock
```

Comment:
Nit: link to https://github.com/giggio/node-chromedriver?tab=readme-ov-file#detect-chromedriver-version

This rocks! Awesome!

Reply:
good suggestion, done!